### DISCUSSION TITLE
Notification Sign Can't Disappear

### DISCUSSION THREAD

**Discussion by SupianIDz** [community/community#112335] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

A week ago I received an email that someone mentioned me on Gist, but when I opened it, it showed 404 https://gist.github.com/no198/0e7b12f252aa5247420c56fb53061fec#gistcomment-4968382

After that, my notifications always showed a sign that there were new unread notifications, but when I checked, there weren't any at all.

How to get rid ?

<img width="250" alt="Screenshot 2024-03-13 at 16 51 48" src="https://github.com/community/community/assets/37969970/b93a28cf-c77e-4541-bd67-8276efd45d20">
<img width="1348" alt="Screenshot 2024-03-13 at 16 55 15" src="https://github.com/community/community/assets/37969970/a4e1a964-9c90-4e0d-8775-9e1f5a5f1b70">

<img width="1710" alt="Screenshot 2024-03-13 at 16 52 08" src="https://github.com/community/community/assets/37969970/1fb1240b-41e8-4b69-ab9a-53202ccc241c">
<img width="1710" alt="Screenshot 2024-03-13 at 16 52 17" src="https://github.com/community/community/assets/37969970/60440041-d5de-43e1-86e0-a16065ef2a1f">

<img width="1710" alt="Screenshot 2024-03-13 at 16 52 31" src="https://github.com/community/community/assets/37969970/53ec1c89-b779-4053-bb20-8ab177d267c6">

**Comment by amosbird** [comment, marked as answer]
`gh api notifications -X PUT -F last_read_at=2024-03-12T00:00:00Z` works.

**Reply by SupianIDz** [reply]
thanks, it works.

**Comment by pirate** [comment]
I also have the same issue, seems like when a repo is deleted they don't clear the mention notifications for it properly.

✅ `gh api notifications -X PUT -F last_read_at=2025-02-22T00:00:00Z` worked for me to fix it.

<img width="1412" alt="Screenshot 2025-02-21 at 3 07 44 PM" src="https://github.com/user-attachments/assets/b9a8e48a-c396-41e2-ac39-efbd9979e580" />
